flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/tiered/shuffle/TieredInternalShuffleMaster.java

```diff
@@ -40,7 +41,9 @@ public class TieredInternalShuffleMaster {
 
     public TieredInternalShuffleMaster(Configuration conf) {
         TieredStorageConfiguration tieredStorageConfiguration =
-                TieredStorageConfiguration.fromConfiguration(conf);
+                TieredStorageConfiguration.builder(
+                                conf.getString(NETWORK_HYBRID_SHUFFLE_REMOTE_STORAGE_BASE_PATH))
+                        .build();
```

Comment:
This change seems belonging to the first commit. Otherwise, there will be a compiling error.